Basic_Concepts/Prototype/README.md

```diff
@@ -82,7 +82,7 @@ A JavaScript function is executed when "something" invokes it (calls it).
         return p1 * p2;              // The function returns the product of p1 and p2
     }
 
-###**JavaScript Function Syntax**  
+### **JavaScript Function Syntax**  
```

Comment:
Heading level 3 `###` doesn't need to be enclosed in `**`, emphasized bold.  
`### JavaScript Function Syntax`